endpoints/schema.yml

```diff
@@ -0,0 +1,109 @@
+paths:
+  /pets:
+    get:
+      parameters:
+        - name: tags
+          in: query
+          description: tags to filter by
+          required: false
+          type: array
+          items:
+            type: string
+        - name: limit
+          in: query
+          description: maximum number of results to return
+          required: false
+          type: integer
+          format: int32
+      responses:
+        "200":
+          description: pet response
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/Pet"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/ErrorModel"
+    post:
+      parameters:
+        - name: pet
+          in: body
+          description: Pet to add to the store
+          required: true
+          schema:
+            $ref: "#/definitions/NewPet"
+      responses:
+        "200":
+          description: pet response
+          schema:
+            $ref: "#/definitions/Pet"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/ErrorModel"
+  "/pets/{id}":
+    get:
+      parameters:
+        - $ref: "#/parameters/PetID"
+      responses:
+        "200":
+          description: pet response
+          schema:
+            $ref: "#/definitions/Pet"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/ErrorModel"
+    delete:
+      parameters:
+        - $ref: "#/parameters/PetID"
+      responses:
+        "204":
+          description: pet deleted
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/ErrorModel"
+
+parameters:
+  PetID:
+    name: id
+    in: path
+    description: ID of pet to fetch
+    required: true
+    type: integer
+    format: int64
+
+definitions:
+  Pet:
+    type: object
+    allOf:
```

Comment:
@lenaw I am not sure if we should document complex tags as this one, as it is not supported by swagger.